corehq/apps/reports/dispatcher.py

```diff
@@ -184,7 +184,7 @@ def navigation_sections(cls, context):
                     else:
                         report_contexts.append({
                             'is_active': report.slug == current_slug,
-                            'url': report.get_url(domain=domain),
+                            'url': report.get_url(domain=domain, request=request),
```

Comment:
Here are the current implementations of get_url:
```
$ ag "def get_url\(" .
corehq/apps/dashboard/models.py
104:    def get_url(self, request):

corehq/apps/receiverwrapper/models.py
251:    def get_url(self, repeate_record):
272:    def get_url(self, repeat_record):

corehq/apps/adm/reports/__init__.py
47:    def get_url(cls, domain=None, render_as=None, **kwargs):

corehq/apps/reports/generic.py
635:    def get_url(cls, domain=None, render_as=None, **kwargs):

corehq/__init__.py
155:            def get_url(cls, domain):

custom/_legacy/pact/management/commands/__init__.py
35:    def get_url(self, url, retry=0):
```
All the standard reports use the generic `get_url` unmodified, it seems, beyond that, there's not really any agreement on arguments.  Passing request in seems reasonable, as it contains a great deal of context.  Does anything besides standard and custom reports use this dispatcher?  (Do user defined report?).